network-api/networkapi/buyersguide/views.py

```diff
@@ -89,7 +89,12 @@ def buyersguide_home(request):
 def category_view(request, categoryname):
```

Comment:
Can we change the parameter name from `categoryname` to `category_slug` while we're here?